mne/viz/ica.py

```diff
@@ -400,7 +400,7 @@ def _plot_ica_sources_evoked(evoked, picks, exclude, title, show, labels=None):
     exclude_labels = list()
     for ii in picks:
         if ii in exclude:
-            line_label = 'ICA %03d' % (ii + 1)
```

Comment:
it was on purpose as channel names start at 1 not 0. Otherwise we write `"ICA #%d" % i`  ie with the # sign.

if you agree please close or suggest something to make it clearer.